tests/driver/api.test.ts

```diff
@@ -276,23 +278,42 @@ describe(`Mongoose Model API level tests`, async () => {
             assert.strictEqual(error!.message, 'aggregate() Not Implemented');
             //----------------//
         });
-        //TODO
-        it.skip('API ops tests Model.applyDefaults()', async () => {
-            Model.applyDefaults();
-            new Product().applyDefaults({name: 'Product 1', price: 10, isCertified: true, category: 'cat 1'});
+        it('API ops tests Model.applyDefaults()', async () => {
+            const modelName = 'User';
```

Comment:
Probably worthwhile to skip `applyDefaults()` too, because that doesn't make any database calls